### DISCUSSION TITLE
Greater control over density of fields in CP

### DISCUSSION THREAD

**Discussion by wesort** [statamic/cms#5740] [Q&A] [answered]
FR: Control for the density of fields

On some projects there are quite a few fields and it'd be nice to have more control over how dense the fields are laid out. Be this by admins (in the fieldset or blueprint) or by users (when editing, similar to "customise columns" on entry listings). This allows more fields to be "packed into the screen" and "above the fold". Honing this authoring experience is one of Statamic's great strengths 💪 

In a naive way, the `padding` on the CP's `.form-group` class could be overridden ([video example](https://app.usebubbles.com/vvqkWsa3p7bSyeXaN81gkd/s-form-group-padding-option)), but I suspect there are smarter ways to go about this in terms of how blueprints / fieldsets and user preferences are implemented.

Examples in the wild include Gmail.
![display density - Gmail](https://user-images.githubusercontent.com/3920126/161563828-2f74fc3a-843f-472b-977c-af4bbb24412e.png)

**Comment by afonic** [comment, marked as answer]
There is a repo for feature requests: https://github.com/statamic/ideas

**Reply by wesort** [reply]
Ah, my mistake - apologies.
